src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfSummaryRequestHandlerTests.cs

```diff
@@ -104,11 +105,14 @@ public async Task HandleAsync_GivenRequest_ShouldCheckOrganisationAccess()
         [Fact]
         public async Task HandleAsync_GivenRequest_SummaryTotalsShouldBeRetrieved()
         {
+            //arrange
+            short currentYear = (short)SystemTime.Now.Year;
+
             //act
             await handler.HandleAsync(request);
 
             //assert
-            A.CallTo(() => evidenceStoredProcedures.GetAatfEvidenceSummaryTotals(request.AatfId, 1))
+            A.CallTo(() => evidenceStoredProcedures.GetAatfEvidenceSummaryTotals(request.AatfId, currentYear))
```

Comment:
update to
A.CallTo(() => evidenceStoredProcedures.GetAatfEvidenceSummaryTotals(request.AatfId, request.ComplianceYear))